src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -238,6 +241,11 @@ def do_fgs():
     with TRACER.start_span("move_to_z_0"):
         yield from bps.abs_set(fgs_motors.z_steps, 0, wait=False)
 
+    # Wait for arming to finish
+    artemis.log.LOGGER.debug("Waiting for arming...")
+    yield from bps.wait("arming")
+    artemis.log.LOGGER.debug("Arming finished")
```

Comment:
Nit: Great use of logging but might as well call these `INFO` rather than `DEBUG`